addons/account/views/report_invoice.xml

```diff
@@ -194,9 +194,12 @@
                                             <td name="td_taxes" t-attf-class="text-end {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }} {{ 'text-nowrap' if len(taxes) &lt; 10 else '' }}">
                                                 <span t-out="taxes" id="line_tax_ids">Tax 15%</span>
                                             </td>
-                                            <td name="td_subtotal" class="text-end o_price_total">
+                                            <td t-if="o.company_price_include == 'tax_excluded'" name="td_subtotal" class="text-end o_price_total">
```

Comment:
so you put the t-if on the td and not on the span because the name was td_subtotal?  As the td will have the same structure I suppose...  (It is basically the price that changes)  @aizu-odoo